src/pointarray.cpp

```diff
@@ -44,8 +44,8 @@ struct OctreeNode
     size_t endIndex;         ///< End index of points in this node
     mutable size_t nextBeginIndex; ///< Next index for incremental rendering
     Imath::Box3f bbox;       ///< Actual bounding box of points in node
-    V3f center;              ///< Centre of the node
-    float radius;            ///< Distance to centre to node edge along an axis
+    V3f center;              ///< center of the node
```

Comment:
Ok, I submit to the American spelling :)

(I do agree it's much better to be consistent here.  American spelling is so ingrained in APIs in general that I've started just using it in code.)